frontend/src/domain/use_cases/missions/editMissionInLocalStore.ts

```diff
@@ -1,67 +1,53 @@
 import { generatePath } from 'react-router'
 
 import { missionsAPI } from '../../../api/missionsAPI'
+import { attachReportingToMissionSliceActions } from '../../../features/missions/MissionForm/AttachReporting/slice'
+import { missionFormsActions } from '../../../features/missions/MissionForm/slice'
 import { removeMissionListener } from '../../../features/missions/MissionForm/sse'
-import { attachReportingToMissionSliceActions } from '../../../features/missions/slice'
+import { missionActions } from '../../../features/missions/slice'
 import { sideWindowActions } from '../../../features/SideWindow/slice'
 import { sideWindowPaths } from '../../entities/sideWindow'
 import { setToast } from '../../shared_slices/Global'
-import { multiMissionsActions } from '../../shared_slices/MultiMissions'
 
-export const editMissionInLocalStore = missionId => async (dispatch, getState) => {
-  const {
-    missionState: { isFormDirty, missionState },
-    multiMissions: { selectedMissions }
-  } = getState()
+export const editMissionInLocalStore = (missionId: number) => async (dispatch, getState) => {
+  const { missions } = getState().missionForms
 
   const missionToEdit = missionsAPI.endpoints.getMission
 
-  try {
-    const missionRequest = dispatch(missionToEdit.initiate(missionId))
-    const missionResponse = await missionRequest.unwrap()
+  if (missions[missionId]) {
+    setMission(dispatch, missions[missionId])
```

Comment:
Un early return pour éviter le `else` ?

```suggestion
  if (missions[missionId]) {
    setMission(dispatch, missions[missionId])
    return
 }
```